### PR TITLE
Update scheduler property docs

### LINKED ISSUE #345: Update scheduler property docs
Account for updates made in: https://github.com/spring-cloud/spring-cloud-scheduler-kubernetes/issues/11

also add section on `SPRING_CLOUD_SCHEDULER_KUBERNETES_NAMESPACE` in `server-deployment.yaml`

### PR DESCRIPTION
**Pull request by chrisjs**
Resolves #345

### CODE REVIEW

**Comment by chrisjs** [comment]
depends on https://github.com/spring-cloud/spring-cloud-scheduler-kubernetes/pull/12

File: spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/configuration.adoc [hunk 1]
```diff
@@ -377,31 +377,184 @@ Request to clean up resources for task execution 1 has been submitted
...
 - name: SPRING_CLOUD_SCHEDULER_KUBERNETES_ENTRY_POINT_STYLE
-  value: 'shell'
-- name: SPRING_CLOUD_DATAFLOW_FEATURES_SCHEDULES_ENABLED
-  value: 'false'
+  value: entryPointStyle
 ----
 ====
 
-See https://github.com/spring-cloud/spring-cloud-scheduler-kubernetes/blob/master/src/main/java/org/springframework/cloud/scheduler/spi/kubernetes/KubernetesSchedulerProperties.java[`KubernetesSchedulerProperties`] for more on the supported options.
+Replace `entryPointStye` with the desired Entry Point Style.
```
Comment (cppwfs) [inline spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/configuration.adoc]: entryPointStyle is misspelled.  Will fix on merge.

File: spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/configuration.adoc [hunk 2]
```diff
@@ -377,31 +377,184 @@ Request to clean up resources for task execution 1 has been submitted
...
+- name: SPRING_CLOUD_SCHEDULER_KUBERNETES_IMAGE_PULL_POLICY
+  value: Always
+----
+====
+
+Replace `Always` with your desired image pull policy.
+
+=== Private Docker Registry
+
+Docker images that are private and require authentication can be pulled by configuring a Secret. First, you must create a secret in the cluster. Follow the https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/[Pull an Image from a Private Registry] guide to create the secret.
```
Comment (cppwfs) [inline spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/configuration.adoc]: Should second  and third `secret` in paragraph be capitalized?

File: spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/configuration.adoc [hunk 3]
```diff
@@ -377,31 +377,184 @@ Request to clean up resources for task execution 1 has been submitted
 
 == Scheduling
 
-Scheduling of tasks is enabled by default in the Spring Cloud Data Flow Kubernetes Server. The following environment variables may be declared as part of the container env section of the server deployment YAML file to customize the defaults if needed:
+This section covers customization of how scheduled tasks are configured. Scheduling of tasks is enabled by default in the Spring Cloud Data Flow Kubernetes Server. Properties are used to influence settings for scheduled tasks and can be configured on a global or per-schedule basis.
```
Comment (cppwfs) [inline spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/configuration.adoc:380]: Replace influence with `establish the`

Comment (chrisjs) [reply spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/configuration.adoc:380]: keeping as that was done in a prior editing pass

File: spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/configuration.adoc (hunk 1 above)
Comment (chrisjs) [reply spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/configuration.adoc]: will fix on push

File: spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/configuration.adoc (hunk 2 above)
Comment (chrisjs) [reply spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/configuration.adoc]: yup, will update on push

**Comment by chrisjs** [comment]
PR comments addressed

**Comment by chrisjs** [comment]
merged in a75dc7c

**Pull request closed without merging**